app/api.py

```diff
@@ -341,6 +343,42 @@ def get(self):
             self, ApiActionLog.READ, len(records), len(notes))
 
 
+class PhotoUpload(utils.BaseHandler):
+    https_required = True
+
+    def post(self):
+        if not (self.auth and self.auth.domain_write_permission):
+            self.info(
+                403,
+                message='Missing or invalid authorization key',
+                style='plain')
+            return
+
+        # Check for empty body
+        if len(self.request.body) < 1:
```

Comment:
Can you write "if not self.request.body" instead? It is preferred per style guide we follow:
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html#True/False_evaluations